src/Symfony/Component/Messenger/ParallelMessageBus.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger;
+
+use Amp\Parallel\Worker\ContextWorkerPool;
```

Comment:
This requires validation and exception suggesting `composer require amp/*` call if not available.

Reply:
is the check in the `Worker` class sufficient?

Reply:
No =)